src/axom/quest/readers/C2CReader.cpp

```diff
@@ -23,6 +28,178 @@ namespace axom
+ */
+size_t next_longest(const std::vector<Segment>& segments)
+{
+  size_t maxIdx = 0;
+  double max_length = 0.;
+  size_t n = segments.size();
+  for(size_t i = 0; i < n; i++)
+  {
+    double next_length = segments[i].next_length[0] + segments[i].next_length[1];
+    if(next_length > max_length)
+    {
+      max_length = next_length;
+      maxIdx = i;
+    }
+  }
+  return maxIdx;
+}
+
+//---------------------------------------------------------------------------
+/*!
+ * \brief Append a set of segments to a mint mesh, filtering out like points.
+ * \param mesh The mint mesh to which we're adding points/cells.
+ * \param S The segments to write.
+ * \param EPS_SQ The squared distance for point matching.
+ */
+static void appendPoints(mint::UnstructuredMesh<mint::SINGLE_SHAPE>* mesh,
+                         std::vector<Segment>& S,
+                         double EPS_SQ)
+{
+  // Check for simple vertex welding opportunities at endpoints of newly interpolated points
+  {
+    int numNodes = mesh->getNumberOfNodes();
+    if(numNodes > 0)  // this is not the first Piece
+    {
+      primal::Point<double, 2> meshPt;
+      // Fix start point if necessary; check against most recently added vertex in mesh
+      mesh->getNode(numNodes - 1, meshPt.data());
+      if(primal::squared_distance(S[0].point, meshPt) < EPS_SQ)
+      {
+        S[0].point = meshPt;
+      }
+
+      // Fix end point if necessary; check against 0th vertex in mesh
+      const int endIdx = S.size() - 1;
+      mesh->getNode(0, meshPt.data());
+      if(primal::squared_distance(S[endIdx].point, meshPt) < EPS_SQ)
+      {
+        S[endIdx].point = meshPt;
+      }
+    }
+    else  // This is the first, and possibly only span, check its endpoint, fix if necessary
+    {
+      int endIdx = S.size() - 1;
+      if(primal::squared_distance(S[0].point, S[endIdx].point) < EPS_SQ)
+      {
+        S[endIdx].point = S[0].point;
+      }
+    }
+  }
+
+  // Add the new points and segments to the mesh, respecting welding checks from previous block
+  {
+    const int startNode = mesh->getNumberOfNodes();
+    const int numNewNodes = S.size();
+    mesh->reserveNodes(startNode + numNewNodes);
+
+    for(int i = 0; i < numNewNodes; ++i)
+    {
+      mesh->appendNode(S[i].point[0], S[i].point[1]);
+    }
+
+    const int startCell = mesh->getNumberOfCells();
+    const int numNewSegments = S.size() - 1;
+    mesh->reserveCells(startCell + numNewSegments);
+    for(int i = 0; i < numNewSegments; ++i)
+    {
+      IndexType seg[2] = {startNode + i, startNode + i + 1};
+      mesh->appendCell(seg, mint::SEGMENT);
+    }
+  }
+}
+
+//---------------------------------------------------------------------------
+#ifdef AXOM_DEBUG_WRITE_LINES
+/*!
+ * \brief Write line segments to a VTK file for visualization.
+ * \param filename The name of the file to write.
+ * \param S The segments to write.
+ */
+static void write_lines(const std::string& filename, const std::vector<Segment>& S)
```

Comment:
`write_lines` and `next_longest` are pothole case while it looks like everything else is camel case. We should be consistent one way or the other.

Reply:
I changed those to camel case.